src/5e-SRD-Features.json

```diff
@@ -48,6 +71,36 @@
     "desc": [
       "Starting at 2nd level, you can throw aside all concern for defense to attack with fierce desperation. When you make your first attack on your turn, you can decide to attack recklessly. Doing so gives you advantage on melee weapon attack rolls using Strength during this turn, but attack rolls against you have advantage until your next turn."
     ],
+    "feature_specific": {
+      "advantages": [
+        {
+          "on": [
+            {
+              "action": "attack",
+              "attack_type": "melee",
+              "attack_modifier": "str",
+              "desc": "Advantage applies to melee attacks with STR modifier only"
+            }
+          ],
+          "duration": "2 turns",
+          "prerequisites": [
+            {
+              "type": "integer",
+              "from": "attack",
```

Comment:
Maybe something like `turn.attacks_made` or `round.attack_number`, or something like that?
Also, ought all prerequisites be booleans (or operations resulting in booleans)? Something like the following?

```json
{
    "type": "equals",
    "description": "attack is the first of the turn",
    "value": {
        "type": "integer",
        "value": "turns.attacks_made",
        "desc": "attacks made so far this turn"
     },
    "target": {
        "type": "integer",
        "value": 0
    }
}
```

Reply:
I like that approach